lib/ace/transport_app.rb

```diff
@@ -137,11 +137,11 @@ def self.init_puppet_target(certname, transport, target)
     end
 
     def scrub_stack_trace(result)
-      if result.dig(:value, '_error', 'details', 'stack_trace')
-        result[:value]['_error']['details'].reject! { |k| k == 'stack_trace' }
+      if result.dig('value', '_error', 'details', 'stack_trace')
```

Comment:
I'm not thrilled about modifying the `result` like this. We do it in bolt-server to so :shrug: